install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -96,9 +96,17 @@ data:
         - {{ "[[ formatDuration .ProxyConfig.ParentShutdownDuration ]]" }}
         - --discoveryAddress
         - {{ "[[ or (index .ObjectMeta.Annotations \"sidecar.istio.io/discoveryAddress\") .ProxyConfig.DiscoveryAddress ]]" }}
-      {{- if .Values.global.zipkinAddress }}
+      {{- if eq .Values.global.proxy.tracer "lightstep" }}
+        - --lightstepAddress
+        - {{ "[[ .ProxyConfig.GetTracing.GetLightstep.GetAddress ]]" }}
+        - --lightstepAccessToken
+        - {{ "[[ .ProxyConfig.GetTracing.GetLightstep.GetAccessToken ]]" }}
+        - --lightstepSecure={{ "[[ .ProxyConfig.GetTracing.GetLightstep.GetSecure ]]" }}
+        - --lightstepCacertPath
+        - {{ "[[ .ProxyConfig.GetTracing.GetLightstep.GetCacertPath ]]" }}
+      {{- else if (eq .Values.global.proxy.tracer "zipkin") or .Values.global.remoteZipkinAddress }}
```

Comment:
Don't think the `or .Values.global.remoteZipkinAddress` clause is required - as the address is only set if the tracer is `zipkin`. The less places remoteZipkinAddress is referenced probably the better - in case it is deprecated in the future.